config/config-template-zap-long.yaml

```diff
@@ -19,16 +19,11 @@ config:
   environ:
     envFile: "path/to/env/file"
 
-  # (Optional) configure to export scan results to OWASP Defect Dojo
-  defectDojo:
-    url: "https://mydefectdojo.example.com/"
-    # ssl: True|False|/path/to/CA/bundle (default: True). for SSL verification
-    ssl: True
-    authorization:
-      username: "rapidast"
-      password: "password"
-        # or
-      token: "abc"
+  scanResultsCollector:
```

Comment:
I renamed it googleCloudStorage, since it's really tied to that at the moment. Unless we use a generic library that is independent from storage type that can authenticate and push.